src/utils.ts

```diff
@@ -238,7 +251,7 @@ export default abstract class BaseCommand extends Command {
     const sequelize = await sequelizeFactory(path)
 
     if (sync) {
-      await sequelize.sync()
+      await sequelize.sync({ force: true })
```

Comment:
Hmmm is this wise? 🤔 This removes all data... But I guess it is used only in `init` command right?

Reply:
You are right. This is only for `init` command